include/config/config_movement.h

```diff
@@ -4,6 +4,25 @@
  * MOVEMENT SETTINGS *
  *********************/
 
+
+// Gives Mario the spinning triple jump found in the beta versions.
+// Most important setting in the file by far
+#define BETA_TRIPLE_JUMP
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
```suggestion

















```

needs to be more obvious its an option, i suggest making `config_beta.h` instead